tpcx_bb/xbb_tools/cluster_startup.py

```diff
@@ -22,7 +22,7 @@
 import dask
 from dask.distributed import Client
 from dask.utils import parse_bytes
-from blazingsql import BlazingContext
+#from blazingsql import BlazingContext
```

Comment:
This would cause a failure downstream if the flag variable you create is ever set to True

Reply:
Will fix once there are CUDA 11 packages for BlazingSQL. This PR can wait until then to merge.